test/test_torch.py

```diff
@@ -12891,7 +12888,9 @@ def test_pow_scalar_overloads_mem_overlap(self, device):
         self.unary_check_input_output_mem_overlap(
             doubles, sz, lambda input, out: torch.pow(42, input, out=out))
 
-    def test_erfinv(self, device):
+    @dtypesIfCUDA(torch.half, torch.float, torch.double)
```

Comment:
it is probably common enough in the future to have a function that returns a device-specific list for these cases.

Reply:
getCUDAFloatTypes()?

Reply:
getFloatTypes('cuda') :).